### PR TITLE
Homework 3 - Task 1,2,3

### PR DESCRIPTION
**Pull request by nguyenphuhao**

### CODE REVIEW

File: src/components/product/productItem/productItemHorizontal/styles.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    shadowOpacity: 0.8,
+    shadowRadius: 2,
+    elevation: 1,
+  },
+  productImg: {
+    alignSelf: 'center',
+    width: '75%',
+    height: '75%',
+  },
+  productName: {},
```
Comment (sharad-epam) [inline src/components/product/productItem/productItemHorizontal/styles.js:66]: remove this empty method

File: src/screens/call/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+const CallScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (sharad-epam) [inline src/screens/call/index.js:5]: get it from styles don't add inline styles

File: src/screens/cart/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+const CartScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (sharad-epam) [inline src/screens/cart/index.js:5]: get it from styles

File: src/screens/email/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+const EmailScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (sharad-epam) [inline src/screens/email/index.js:5]: get it from styles

File: src/screens/profile/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+const ProfileScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (sharad-epam) [inline src/screens/profile/index.js:5]: get it from styles

File: src/screens/share/index.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+const ShareScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (sharad-epam) [inline src/screens/share/index.js:5]: get it from styles

File: src/screens/wishlist/index.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+const WishListScreen = () => {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```
Comment (sharad-epam) [inline src/screens/wishlist/index.js:5]: get it from styles

File: src/components/product/productItem/productItemHorizontal/styles.js (hunk 1 above)
Comment (nguyenphuhao) [reply src/components/product/productItem/productItemHorizontal/styles.js:66]: Resolved in Homework 5

File: src/screens/email/index.js (hunk 4 above)
Comment (nguyenphuhao) [reply src/screens/email/index.js:5]: Resolved in Homework 5

File: src/screens/share/index.js (hunk 6 above)
Comment (nguyenphuhao) [reply src/screens/share/index.js:5]: Resolved in Homework 5

File: src/screens/wishlist/index.js (hunk 7 above)
Comment (nguyenphuhao) [reply src/screens/wishlist/index.js:5]: Resolved in Homework 5

File: src/screens/profile/index.js (hunk 5 above)
Comment (nguyenphuhao) [reply src/screens/profile/index.js:5]: Resolved in Homework 5

File: src/screens/cart/index.js (hunk 3 above)
Comment (nguyenphuhao) [reply src/screens/cart/index.js:5]: Resolved in Homework 5

File: src/screens/call/index.js (hunk 2 above)
Comment (nguyenphuhao) [reply src/screens/call/index.js:5]: Resolved in Homework 5

**Pull request merged**